config/webpack/loaders/devSiteExample.js

```diff
@@ -5,30 +5,53 @@ const startCase = require('lodash.startcase');
  * Generate the example with the supplied file.
  * Don't use an arrow function
  */
-const loader = async function loader() {
+const loader = async function loader(content) {
   const callback = this.async();
 
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
+  let cssFileName;
+
+  if (content.includes('.scss') || content.includes('.css')) {
+    cssFileName = content.slice(
```

Comment:
Much cleaner!
![](https://media.giphy.com/media/5DQdk5oZzNgGc/giphy.gif)

Reply:
Lets make this a separate function so we can jest test it easily. I'd like to have some examples of what kinds of strings work here and what doesn't.

Reply:
Personally i'd use a regex and a capture group to grab the first file path that ended in css
`^.*\'(.*css)\'`

A playground where you can try out the regex
https://regex101.com/r/QPsqRD/1/

Reply:
Made a separate file [here](https://github.com/cerner/terra-dev-site/pull/281/commits/f4ff529e1f5620f5daf36cf5aaf4f9acab585291), and make a few jest tests [here](https://github.com/cerner/terra-dev-site/pull/281/commits/10f422264a2dd500cfd617de9a329d7e8667e1f7). Currently it checks for a shortened file path, an explicit file path, a lack of file type, and the file name containing css but there isn't a file type.